telos-frontend/src/components/calendar/DashboardCalendar.jsx

```diff
@@ -1,36 +1,103 @@
 import { Calendar, momentLocalizer } from 'react-big-calendar';
+import { useState } from 'react';
 import moment from 'moment';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
 import 'react-big-calendar/lib/css/react-big-calendar.css';
 import './react-big-calendar.css';
 
 const localizer = momentLocalizer(moment);
 
-const events = [
-  {
-    id: 14,
-    title: 'Today',
-    start: new Date(new Date().setHours(new Date().getHours() - 3)),
-    end: new Date(new Date().setHours(new Date().getHours() + 1)),
-  },
-  {
-    id: 23,
-    title: 'Go to the gym 💪',
-    start: new Date(new Date().setHours(new Date().getHours() - 3)),
-    end: new Date(new Date().setHours(new Date().getHours() + 5)),
-  },
-];
-
-const DashboardCalendar = () => (
-  <div>
-    <Calendar
-      selectable
-      localizer={localizer}
-      events={events}
-      startAccessor="start"
-      endAccessor="end"
-      style={{ height: window.screen.height - 110, width: '100%' }}
-    />
-  </div>
-);
+const sendAddEventRequest = () => {
+  const a = 'temp';
+  return a;
+};
+
+const getRandomInt = (max) => Math.floor(Math.random() * Math.floor(max));
+
+const DashboardCalendar = () => {
+  const [open, setOpen] = useState(false);
+  const [selectedId, setSelectedId] = useState(0);
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const [events, setEvents] = useState([
+    {
+      id: 14,
+      title: 'Today',
+      start: new Date(new Date().setHours(new Date().getHours() - 3)),
+      end: new Date(new Date().setHours(new Date().getHours() + 1)),
+    },
+    {
+      id: 23,
+      title: 'Go to the gym 💪',
+      start: new Date(new Date().setHours(new Date().getHours() - 3)),
+      end: new Date(new Date().setHours(new Date().getHours() + 5)),
+    },
+  ]);
+
+  return (
+    <div>
+      <Calendar
+        selectable
+        localizer={localizer}
+        events={events}
+        startAccessor="start"
+        endAccessor="end"
+        style={{ height: window.screen.height - 110, width: '100%' }}
+        onSelectEvent={(event) => {
+          setSelectedId(event.id);
+          handleClickOpen();
+        }}
+        onSelectSlot={({ start, end }) => {
+          // eslint-disable-next-line no-alert
+          const title = window.prompt('New Event name');
```

Comment:
I noticed you've added Material UI for the delete event, but not for this add event. How come?

Reply:
Also, please attach screenshots though so that the others can easily view what you've developed :)

Reply:
Yup, all done.
- Added material ui to add event.
- Added screenshots.

Reply:
Awesome!